pkg/controller/config/config_controller.go

```diff
@@ -304,6 +321,14 @@ func (r *ReconcileConfig) removeStaleExpectations(stale *watch.Set) {
 	}
 }
 
+func (r *ReconcileConfig) skipExcludedNamespace(namespace string) bool {
+	if r.processExcluder != nil {
```

Comment:
I'm split here. The nil check seems like it would make testing easier, but it could help flush out bugs if we throw a nil-reference-error every time we run against an uninitialized manager.

Reply:
removed the nil check for now